karate-tests/src/test/java/patients/healthcareWorker/updatePatientDetails/replace.feature

```diff
@@ -226,4 +226,72 @@ Feature: Patch patient - Replace data
       * status 400
       * def display = 'Patient cannot perform this action'
       * def diagnostics = "Invalid update with error - interpreterRequired cannot be removed"
-      * match response == read('classpath:mocks/stubs/errorResponses/INVALID_UPDATE.json')
\ No newline at end of file
+      * match response == read('classpath:mocks/stubs/errorResponses/INVALID_UPDATE.json')
+      
+Scenario: Healthcare worker can't remove usual name and DOB
+  * def expectedResponse = read('classpath:mocks/stubs/errorResponses/FORBIDDEN_UPDATE.json')
+  * def nhsNumber = '9733162043'
+  * configure headers = call read('classpath:auth/auth-headers.js') 
+  * path 'Patient', nhsNumber
+  * method get
+  * status 200
+  * def originalVersion = parseInt(response.meta.versionId)
+  * def nameIndex = response.name.findIndex(x => x.use == 'usual')
+  * def namePathToRemove = "/name/"+ nameIndex 
+  * def nameDetails = response.name.find(x => x.use == 'usual')
```

Comment:
Nit pick: this makes it clearer what's important about this name - we've pick a usual name
```suggestion
  * def usualNameIndex = response.name.findIndex(x => x.use == 'usual')
  * def pathToUsualName = "/name/"+ nameIndex 
  * def usualNameDetails = response.name.find(x => x.use == 'usual')
```